project.clj

```diff
@@ -39,9 +39,7 @@
                  [colorize "0.1.1" :exclusions [org.clojure/clojure]] ; string output with ANSI color codes (for logging)
                  [com.amazon.redshift/redshift-jdbc41 "1.2.8.1005"]   ; Redshift JDBC driver
                  [com.cemerick/friend "0.2.3"                         ; auth library
-                  :exclusions [commons-codec
-                               org.apache.httpcomponents/httpclient
```

Comment:
Just so I understand what's going on here. Do the drivers depend on having `org.apache.httpcomponents/httpclient` around?